code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -670,6 +670,102 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	drop_sound = 'sound/items/drop/accessory.ogg'
 	pickup_sound = 'sound/items/pickup/accessory.ogg'
 
+/obj/item/flame/lighter/zippo/dominia
+	name = "\improper Dominian Zippo lighter"
+	desc = "A zippo lighter with a depiction of the Imperial standard of Dominia."
+	desc_fluff = "While never officially endorsed by the Emperor, lighters featuring a stylized Imperial standard are a common sight throughout the Empire. Due to the simplicity with which the standard can be recreated, these lighters are found even in the poorer frontier regions in the Empire and are commonly seen as a sign of patriotism."
+	icon_state = "dominiazippo"
+	item_state = "dominiazippo"
+	activation_sound = 'sound/items/cigs_lighters/zippo_on.ogg'
+	deactivation_sound = 'sound/items/cigs_lighters/zippo_off.ogg'
+	drop_sound = 'sound/items/drop/accessory.ogg'
+	pickup_sound = 'sound/items/pickup/accessory.ogg'
```

Comment:
you don't need to copypasta this since it's already defined in the master zippo

backspace all of this from all the lighters you've added

Reply:
All done, chief.